modules/vba_documents/app/controllers/vba_documents/docs/v2/api_controller.rb

```diff
@@ -8,23 +8,7 @@ class ApiController < ApplicationController
         include Swagger::Blocks
 
         SWAGGERED_CLASSES = [
-          VBADocuments::DocumentUpload::FailureSwagger,
-          VBADocuments::DocumentUpload::V2::MetadataSwagger,
-          VBADocuments::DocumentUpload::StatusSwagger,
-          VBADocuments::DocumentUpload::StatusReportSwagger,
-          VBADocuments::DocumentUpload::StatusGuidListSwagger,
-          VBADocuments::DocumentUpload::V2::WebhookSwagger,
-          VBADocuments::DocumentUpload::V2::WebhookResponseSwagger,
-          VBADocuments::DocumentUpload::SubmissionSwagger,
-          VBADocuments::DocumentUpload::UploadSwagger,
-          VBADocuments::DocumentUpload::V1::PdfUploadAttributesSwagger,
-          VBADocuments::DocumentUpload::V1::PdfDimensionAttributesSwagger,
-          VBADocuments::DocumentUpload::V1::StatusAttributesSwagger,
-          VBADocuments::DocumentUpload::V2::UploadAttributesSwagger,
-          VBADocuments::V2::ControllerSwagger,
-          VBADocuments::V2::ErrorModelSwagger,
-          VBADocuments::V2::SecuritySchemeSwagger,
-          VBADocuments::V2::SwaggerRoot
+          VBADocuments::V2::ControllerSwagger
```

Comment:
To avoid hitting the Danger line limit, I had to retain `VBADocuments::V2::ControllerSwagger` for now. It will be removed in the next PR.